File: exemples/bonjour.f90
program bonjour
use mpi
integer p, np, ierr
call MPI_INIT(ierr)
call MPI_COMM_SIZE(MPI_COMM_WORLD, np, ierr)
call MPI_COMM_RANK(MPI_COMM_WORLD, p, ierr)
print *, 'Ici le processus ',p, ' de ',np
call MPI_FINALIZE(ierr)
end

File: demos/bonjour.c
#include <stdio.h>
#include <mpi.h>

int main(int argc,char *argv[]) {
  int p, np;
  MPI_Init(&argc,&argv);

  MPI_Comm_rank(MPI_COMM_WORLD, &p);
  MPI_Comm_size(MPI_COMM_WORLD, &np);
  printf( "Ici le processus %d de %d\n", p, np );

  MPI_Finalize();
  return 0;
}
